docs/3.0.0-beta.x/getting-started/troubleshooting.md

```diff
@@ -35,9 +35,9 @@ Generally your "flow" of development would follow the following path:
 
 At this time and in the future there is no plan to allow model creating or updating while in a production environment, and there is currently no plans to move model settings into the database. There is no known nor recommended workarounds for this.
 
-### Does Strapi handle deploying or migrating of content?
+### Does Strapi handle deploying or migrating of content
```

Comment:
The stable version will include a migration tool of the settings to facilitate the switch between environments.

Reply:
I'm going to re-target this PR to the documentation branch. I'll create a new PR for the changes reflected in the rc releases